DscResources/MSFT_xMySqlDatabase/MSFT_xMySqlDatabase.schema.mof

```diff
@@ -1,9 +1,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlDatabase")] 
 class MSFT_xMySqlDatabase : OMI_BaseResource
 {
-    [Key, Description("Name of the database.")] String Name;
-    [Required, Description("Should the database be present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-    [Required, Description("The root credential that is used to install MySql server."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
+    [Key, Description("Name of the database.")] String DatabaseName;
```

Comment:
I m not sure if I follow. Why would it be ok to break existing resources that depend on this resource or existing configurations unless it is really needed. I believe the property "Name" for the SQLDatabase resource is descriptive enough and it would mean "DatabaseName". I would revert this change.